docker-compose/docker-compose-dev.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  todo-app-development:
+    image: mod5:dev
+    environment:
+      TRELLO_KEY: <SECRET>
+      TRELLO_TKN: <SECRET>
+      TRELLO_BOARD: <SECRET>
```

Comment:
Do you expect developers to update this file and not commit it?

What you want is
```
    env_file:
      - ../.env
```

But actually if you're binding the whole directory, then that includes the .env file anyway so you don't need this.

(.dockerignore doesn't affect volumes, just building images)

Reply:
Yep, simplified this.